index.bs

```diff
@@ -4524,7 +4570,7 @@ enum MLPaddingMode {
 
 dictionary MLPadOptions {
   MLPaddingMode mode = "constant";
-  float value = 0;
+  MLFiniteNumber value = 0;
```

Comment:
Padding should just permit any constant value through, consistently with `constant`, unless you encountered a good reason. People may want to fill the borders with infinity or even NaN, and once the MLNumber is cast to the tensor data type, the actual padding operation is just a dumb bit fill (no element math).

Reply:
+1 to be consistent with `constant`.

Reply:
Done in a5f299d - lets us remove MLFiniteNumber entirely. \o/